core/properties/propertyManager.js

```diff
@@ -20,6 +20,19 @@ export class PropertyManager {
     }
   }
 
+  setItemProperties(property, newPropertyValue) {
+    // console.log('Property Manager - setItemProperties');
+    for (let i = 0; i < this.core.scene.selectionSet.length; i++) {
+      if (typeof(this.core.scene.items[this.core.scene.selectionSet[i]][property]) === 'number' && isNaN(newPropertyValue)) {
+        this.core.notify('Incorrect input type');
```

Comment:
Would we want to break the loop here to save multiple error messages?

would it be possible catch more type errors?

if typeof(...selectionSet[i][property] !== typeof(property))